### PR TITLE
Added home page with fb login

### PR DESCRIPTION
**Pull request by nicofh**
**Description:**
Created home page
User is required to login to access features

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -13,3 +13,76 @@
  *= require_tree .
  *= require_self
  */
+html, body {
```
Comment (fedeagripa) [inline app/assets/stylesheets/application.css:16]: Please order alphabetically inside each definition and use rem instead of px by default (you can override the default rem size to 10 px with body{font-size:62.5%} for simpler maths)

File: app/views/devise/sessions/new.html.haml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+= image_tag image_path("smilies_big.svg"), |
```
Comment (fedeagripa) [inline app/views/devise/sessions/new.html.haml]: I think you don't need the .html in the file name

File: app/views/devise/sessions/new.html.haml [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+  class: "smilies-big"                        |
```
Comment (fedeagripa) [inline app/views/devise/sessions/new.html.haml]: Are the | intended to be here?

File: app/views/targets/_facebook_logout.html.haml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+:javascript
```
Comment (fedeagripa) [inline app/views/targets/_facebook_logout.html.haml]: Same here for the .html

File: app/views/targets/_facebook_logout.html.haml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    FB.init({
+      appId      : '1653169284778740',
+      cookie     : true,
+      xfbml      : true,
+      version    : 'v2.12'
+    });
+    FB.AppEvents.logPageView();
+  };
+
+  (function(d, s, id){
```
Comment (fedeagripa) [inline app/views/targets/_facebook_logout.html.haml]: can you refactor the variable names? is really hard to follow the method as it is

File: app/views/targets/_facebook_logout.html.haml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+     js = d.createElement(s); js.id = id;
+     js.src = "https://connect.facebook.net/en_US/sdk.js";
+     fjs.parentNode.insertBefore(js, fjs);
+   }(document, 'script', 'facebook-jssdk'));
+.fb-logout-block
+  - unless current_user
+    = link_to "Sign in with Facebook", user_facebook_omniauth_authorize_path, class: 'fb'
+  - else
+    .fb
+      = current_user[:name]
```
Comment (fedeagripa) [inline app/views/targets/_facebook_logout.html.haml]: current_user.name doesn't work?

**Review by fedeagripa — COMMENTED** [review]
Can you exclude the .DS_Store for the pr please?

File: app/views/targets/_facebook_logout.haml [hunk 7]
```diff
@@ -0,0 +1,25 @@
+:javascript
+  window.fbAsyncInit = function() {
+    FB.init({
+      appId      : '1653169284778740',
```
Comment (oscarsiniscalchi) [inline app/views/targets/_facebook_logout.haml]: move this id to an ENV variable
something like `ENV.fetch('FACEBOOK_APP_ID')`
then you can have the application.yml define this for each environment like staging or localhost

**Review by fedeagripa — COMMENTED** [review]
You are missing some px to rem changes

File: config/routes.rb [hunk 8]
```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  root to: 'targets#index'
-  resources :targets, only: %i[new create index]
   devise_for :users, controllers: { omniauth_callbacks: 'users/omniauth_callbacks' }
+
+  authenticated :user do
+    root to: 'targets#index'
+  end
```
Comment (oscarsiniscalchi) [inline config/routes.rb:7]: empty line between methods

File: config/routes.rb [hunk 9]
```diff
@@ -1,6 +1,12 @@
...
+  devise_scope :user do
+    root to: 'devise/sessions#new'
+  end
```
Comment (oscarsiniscalchi) [inline config/routes.rb]: add empty line

**Review by oscarsiniscalchi — CHANGES_REQUESTED** [review]

**Review by fedeagripa — APPROVED** [review]
